### PR TITLE
Fix #15: Clear all option

### LINKED ISSUE #15: "Clear All" option
"Clear All" option would be rad. It would delete all current labels in a repo.

### PR DESCRIPTION
**Pull request by Jameskmonger**

### CODE REVIEW

**Comment by himynameisdave** [comment]
This is super rad, thanks for doing this man :+1: As soon as I'm done with [the refactor I'm working on](https://github.com/himynameisdave/git-labelmaker/issues/9) I'll merge this guy in,

File: index.js [hunk 1]
```diff
@@ -99,6 +99,18 @@ const doRemovePrompts = ( token, repo ) => {
         });
       };
 
+const removeAllLabels = ( token, repo ) => {
+  octonode.client(token).get('/repos/' + repo + '/labels', (e, status, body) => {
+    let allLabels = body.map((label) => {
+      return { name: label.name, color: label.color };
+    });
+
```
Comment (himynameisdave) [inline index.js:38]: @Jameskmonger my only bit of code review here would be to add a conditional to see if there are even any labels to remove. If there aren't any, we can just `console.log("No labels to remove")` and not run `gitLabel.remove`, which is going to try to make that external API call to delete 0 labels.

Thanks again for contributing! :smile:

**Comment by Jameskmonger** [comment]
@himynameisdave Done!

**Comment by Jameskmonger** [comment]
@himynameisdave I've made my master branch up to date with your master branch so this can be automatically merged in.

**Comment by himynameisdave** [comment]
@Jameskmonger looks like there is only one file changed... sorry I just did a huge refactor and a lot of stuff got moved around, I wasn't expecting this feature to be ready so soon.

**Comment by Jameskmonger** [comment]
I will sort the changes out and reopen this @himynameisdave :smile:

**Comment by himynameisdave** [comment]
Cool thanks man, I appreciate the effort!

**Comment by Jameskmonger** [comment]
Reopened this @himynameisdave

**Pull request closed without merging**